README.md

```diff
@@ -277,13 +347,17 @@ You can add trusted facts to the factsets by creating a new section called trust
 
 Notice that the `extensions` part is implied. The first fact in that example translates to `$trusted['extensions']['pp_role']` in Puppet code.
 
-### Hiera Data
+### Hiera
 
-If you have hiera data inside your controlrepo (or somewhere else) Onceover can be configured to use it. It is however worth noting the the `hiera.yaml` file that you currently use may not be applicable for testing right away. For example; if you are using `hiera-eyaml` I recommend creating a `hiera.yaml` purely for testing that simply uses the `yaml` backend, meaning that you don't need to provide the private keys to the testing machines.
+If you have hiera data inside your controlrepo (or somewhere else) `onceover` can be configured to use it.
+It is however worth noting the `hiera.yaml` file that you currently use may not be applicable for testing right away.
+For example; if you are using `hiera-eyaml` I recommend creating a `hiera.yaml` purely for testing that simply uses the `yaml` backend, meaning that you don't need to provide the private keys to the testing machines.
 
-It is also worth noting that any hiera hierarchies that are based on custom facts will not work unless those facts are part of your factsets. Trusted facts will also not work at all as the catalogs are being compiled without the node's certificate. In these instances it may be worth creating a hierarchy level that simply includes dummy data for testing purposes in order to avoid hiera lookup errors.
+It is also worth noting that any hiera hierarchies that are based on custom facts will not work unless those facts are part of your factsets.
+Trusted facts will also not work at all as the catalogs are being compiled without the node's certificate.
```

Comment:
Given the section `Trusted Facts` above, is this still true?

Reply:
@alexjfisher I dont know. I only rework from "Quick start" to "Configuration" as specified in description.
BTW, this PR remains relevant and if `Trusted Facts` are outdated, it could be reworked later.

Reply:
ok.  I was thinking about opening a PR myself (if I'm right about this statement being out of date), but then saw this already open.  I agree that it's not a blocker for this PR as a whole and can submit the change separately.

Reply:
Please note that #278 adds a section to `README.md` according to the feature provided in this PR (ie. rspec tags support).
Maybe you could provide your modifications on top of these commits to prevent from merge conflict.

@dylanratcliffe As you were interested in this PR, I should help us to improve `onceover` easily if you could merge both PR, this one and #278 . Thanks